arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -220,7 +218,25 @@
               "$ref": "#/definitions/CognitiveServicesAccount"
             },
             "examples": {
-              "application/json": "{\n  “id”: “/subscriptions/{id}/resourceGroups/{group}/providers/{rpns}/{type}/{name}”,\n  “name”: “{name}”,\n  “type”: “Microsoft.CognitiveSerices/accounts”,\n  \"location\": \"North US\",\n  \"tags\": {\n    \"key1\": \"value 1\",\n    \"key2\": \"value 2\"\n   },\n  \"kind\" : \"resource kind\",\n   \"sku\": {\n\t    \"name\": \"sku code\"\n\t} \n  “etag”: “00000000-0000-0000-0000-000000000000”,\n  \"properties\": { }\n}"
+              "application/json": {
+                "id": "/subscriptions/f9b96b36-1f5e-4021-8959-51527e26e6d3/resourceGroups/felixwa-01/providers/Microsoft.CognitiveServices/accounts/felixwatest1",
```

Comment:
Also you are missing example for PUT/Patch/delete. Consider adding example for other operations as well.